### DISCUSSION TITLE
How to do quantization for pretrained model?

### DISCUSSION THREAD

**Discussion by EvelinaAleksiutenko** [NVIDIA-NeMo/NeMo#4739] [General]
I need to do quantization for the pretrained model. After reading the [tutorial]([Punctuation_and_Capitalization.ipynb](https://github.com/NVIDIA/NeMo/blob/main/tutorials/nlp/Punctuation_and_Capitalization.ipynb))I try to run the code 
snippet:
```model = nemo_nlp.models.PunctuationCapitalizationModel.from_pretrained("punctuation_en_distilbert")```. The model was loaded correctly. 
```quantized_model = torch.quantization.quantize_dynamic(model, {torch.nn.Linear}, dtype=torch.qint8)```
**Here I faced an error:**
**NotImplementedError: object proxy must define __deepcopy__()**

How can I fix this? How can I actually do quantization for NLP models from NeMo?

**Comment by TheFunyBunky** [comment]
Having the same issue
